dowhy/causal_refuters/bootstrap_refuter.py

```diff
@@ -66,3 +131,34 @@ def refute_estimate(self, *args, **kwargs):
 
         return refute
 
+    def choose_desired_variables(self):
+        '''
+            This method provides a way to choose the confounders whose values we wish to
+            modify for finding its effect on the ability of the treatment to affect the outcome.
+        '''
+        if self._required_variables is None:
+            self._chosen_variables = None
+        elif type(self._required_variables) is int:
+            # Shuffle the confounders 
+            random.shuffle(self._target_estimand.backdoor_variables)
+            self._chosen_variables = self._target_estimand.backdoor_variables[:self._required_variables]
+        elif type(self._required_variables) is list:
+            self._chosen_variables = self._required_variables
+            
+
+        if self._chosen_variables is None:
+            self.logger.info("INFO: There are no chosen variables")
+        else:    
+            self.logger.info("INFO: The chosen variables are: " +
+                            ",".join(self._chosen_variables))
+
+    def get_mask_variables(self):
+        '''
+            This function helps to create a mask that sets false for all values that have a value
+            smaller than the noise level set by the user.
+        '''
+        # Sample a uniform distribution for each data point
+        probs = np.random.uniform(0,1, self._sample_size)

```

Comment:
This is a strange construction, because a user may have thought that the noise variable is standard deviation. What if a user provides noise>1?
The add unobserved confounder refuter asks for the probability of flipping if it is binary variable. At the risk of introducing too many parameters, it may be useful to introduce a separate parameter for flipping probability of a binary variable (and maybe set it to the noise parameter's value by default)

Reply:
That sounds like a good idea.